src/js/tabs/configuration.js

```diff
@@ -628,6 +628,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             $('div.cycles').show();
         }
 
+        if(semver.gte(CONFIG.apiVersion, "1.37.0") || !isExpertModeEnabled()) {
```

Comment:
We should probably also disable this switch when expert mode is not on - otherwise users of older firmware versions can enable it if they accidentally enable expert mode, and then they won't easily find where to disable it.

Reply:
in newer firmware versions or when expertmode isnt enabled the checkbox now will get hidden and enabled.
having it enabled is the betaflight default settings and the save setting